src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,39 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+import './TodoInfo.scss';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
```

Comment:
in destruction you need to get " id " and (add data-id={todo.id} attribute to each .TodoInfo).